jupyterhub/schema.yaml

```diff
@@ -1259,6 +1259,14 @@ properties:
           Configuration for a k8s ServiceAccount dedicated for use by the
           specific pod which this configuration is nested under.
         properties:
+          create:
+            type: boolean
```

Comment:
```suggestion
            type: boolean
            additionalProperties: false
```

Reply:
I didn't add that because it's not there for any other booleans.